src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1,9 @@
+# Safe, no deserialization, only basic types
+org.jfrog.build.client.DeployDetails
+
+# Safe, only basic types. Classes below XStream annotations, so it's supposed to be serialized
+org.jfrog.build.api.Artifact
+org.jfrog.build.api.Dependency
+
+# XStream
+com.google.common.collect.ArrayListMultimap
```

Comment:
Is there a matching core PR for this one?

Reply:
Will submit soon